examples/lb/listener/base-network.tf

```diff
@@ -0,0 +1 @@
+../../networking/network/main.tf
```

Comment:
why do not use networking/main.tf?

Reply:
too many resources that we do not need like router and ext. but if u want i will